p2p/discover/udp.go

```diff
@@ -529,9 +521,6 @@ func (req *ping) handle(t *udp, from *net.UDPAddr, fromID NodeID, mac []byte) er
 	if expired(req.Expiration) {
 		return errExpired
 	}
-	if req.Version != Version {
-		return errBadVersion
-	}
```

Comment:
How does this help during protocol updates? By the newer nodes potentially trying to parse the older nodes' packets? Wouldn't there be problems the other way around when only a few older nodes are left and they can't discover an more (although it's debatable, that such a scenario would appear anyway during any update).